docs/form-field.md

```diff
@@ -0,0 +1,60 @@
+<link href="style.css" rel="stylesheet"></link> 
+
+# PsFormField <a name="PsFormField"></a>
+`<ps-form-field>` extends the [MatFormFieldComponent](https://material.angular.io/components/form-field/overview) from Angular Material with features like:
+* Label handling (labels can be extracted from FormControl informations for example)
+* Handling controls with no `formControlName` given
+
+---
+
+  ## API <a name="PsFormFieldApi"></a>   
+  ### Import <a name="PsFormFieldImport"></a>
+  ```ts | js
+    import { PsFormFieldComponent } from '@prosoft/components/form-field'
+  ```
+
+---
+
+  ## PsFormFieldComponent <a name="PsFormFieldComponent"></a>
+  ### Properties <a name="PsFormFieldComponentProperties"></a>
+  | Name                                 | Description
+  | ------------------------------------ | -------------
+  | `createLabel: boolean`               | If `true` a label will be created automatically based on the information of the given FormControl.

```

Comment:
here it would be useful to show the default value. This should probably be added to every doc file.